lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/classloader/ClassLoaderSafeConnectorMetadata.java

```diff
@@ -410,9 +411,15 @@ public void renameColumn(ConnectorSession session, ConnectorTableHandle tableHan
 
     @Override
     public void dropColumn(ConnectorSession session, ConnectorTableHandle tableHandle, ColumnHandle column)
+    {
+        dropColumn(session, tableHandle, column, ImmutableList.of());
```

Comment:
ClassLoaderSafe must always delegate directly.

(the underlying implementation may still implement the old API, but not the new one; ClassLoaderSafe wrapper should not change the call routing)